app/views/users/index.html.erb

```diff
@@ -1 +1,9 @@
-<h1>Here is a list of all users<h1>
\ No newline at end of file
+<div class='flex flex-col gap-8 mt-8'>
+<% @users.each do |user| %>
+    <div class='flex gap-8 m-1'>
+      <img src="<%= user.photo_url%>" alt="<%= user.name%>" class='w-32 h-32 rounded-full' />
+      <h2 class=' text-3xl'> <%= link_to user.name, user_path(user)  %> </h2>
+      <p> Number of posts: <strong><%= user.posts_counter %></strong></p>
+    </div>
+<% end %>
+ </div> 
```

Comment:
- [OPTIONAL] Great job so far 👏🏻  However You may use Partial templates eg:  `app/views/users/_user.html.erb`, as a better practice instead of using `@objects.each`. Partial templates (partials) are a way of breaking the rendering process into more manageable chunks. Partials allow you to extract pieces of code from your templates to separate files and also reuse them throughout your templates. For more documentation visit this [link](https://riptutorial.com/ruby-on-rails/example/2875/partials)



```ruby
    # app/views/users/_user.html.erb

     #   HTML Code Here displaying
     #   user.photo, user.name and user.posts_counter

    # app/views/users/index.html.erb

    <%= render @users %>
```